templates/search/blocks/search_results__list-card--grid.html

```diff
@@ -30,88 +30,26 @@ <h4 class="tna-heading-s">
             <a href="{% if form.group.value == bucketkeys.ARCHIVE.value %}{% record_url record level_or_archive='Archive' form_group=form.group.value %}{% else %}{% record_url record level_or_archive=record.level form_group=form.group.value %}{% endif %}" class="search-results__list-card-link" data-link="{{ record.reference_prefixed_summary_title }}" data-link-type="Search results list" search-bucket="{{ buckets.current.label }}">
                 {% if record.reference_number %}<span class="sr-only">{{ record.reference_number }}</span>{% endif %}
                 {{ record.summary_title }}
+
+                {# API data required before this can be properly connected to it #}
+                (Type)
             </a>
         </h4>
         <p class="search-results__list-card-description">
             {{ record.listing_description }}
         </p>
-        <dl class="tna-dl tna-dl--plain tna-dl--icon-padding tna-!--margin-bottom-s">
-            {% if record.held_by %}
-                <dt>
-                    <i class="fa-solid fa-landmark"></i>
-                    Held by
-                </dt>
-                <dd>{{ record.held_by }}</dd>
-            {% endif %}
-
+        <dl class="tna-dl tna-dl--plain tna-!--margin-bottom-s">
             {% if record.date_created %}
-                <dt>
-                    <i class="fa-solid fa-calendar"></i>
-                    Date
-                </dt>
```

Comment:
same, please reinstate this as commented inside new dl, for the purpose of tna,non-tna buckets